src/pizzaOrder/Order.java

```diff
@@ -0,0 +1,60 @@
+package pizzaOrder;
+
+import java.util.List;
+
+public class Order {
+
+    private static int orderId = 10000;
+    private static int customerId = 0;
+    private List<Pizza> items;
+
+    public Order() {
+        orderId++;
+        customerId++;
+    }
+
+    public Order(List<Pizza> items) {
+        orderId++;
+        customerId++;
+        checkPizzaName(items);
+        this.items = items;
+
+    }
+
+    private void checkPizzaName(List<Pizza> items) {
```

Comment:
This method does not belong to the Order class please move it to Pizza class

Reply:
Done